doc/misc/npm-orgs.md

```diff
@@ -0,0 +1,70 @@
+npm-orgs(7) -- Working with Teams & Orgs
+========================================
+
+## DESCRIPTION
+
+There are three levels of org users:
+1. Super admin, controls billing & adding people to the org.
```

Comment:
I think you need to add a space here between the line above and the list of numbers for it to render correctly (click the "view" button to preview it)